recipes/dnp-binstrings/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+svn checkout https://github.com/seqan/seqan/trunk/include >/dev/null 2>&1
```

Comment:
This is discouraged to do. Instead you could depend on the seqan library `seqan-library`.

Reply:
I agree. Is there some documentation on a list of available dependencies and their proper names that I could consult. I am not sure that I have very clear understanding how to reference the dependencies.  Thank you for the comments.

Reply:
You can search here: https://anaconda.org/bgruening/dashboard

Everything which is in the conda-forge channel and the Bioconda channel.

Reply:
Thank you, I found it.  I will change  recipes , test locally and will do a new push.  

I have another question, if it is appropriate here.  I did my local testing with bootstrap without docker and it was a success. Now with  CircleCI through github  I have a fail. The reason of fail is:

gcc -std=c++11 -std=c++14 -pthread -I include/ binstrings.cpp -lstdc++ -lrt -lm -o binstrings  
gcc: error: unrecognized command line option ‘-std=c++14’

I am not sure that I will be able to do testing on Ubuntu with docker (short of  resources). It would be helpful if I had a hint what should be changed - why  the gcc -std=c++14  option is not recognized (seems that -std=c++11 passed)  in the test docker environment , what dependency / syntax  I should use in the recipes? 

Many thanks in advance.

Reply:
We are currently building with older compilers by default, and those are not knowing about this option. Is c++14 required by your pkg? If so please add this file to your pkgs https://github.com/bioconda/bioconda-recipes/blob/master/recipes/kmc/conda_build_config.yaml.

It will overwrite the defaults and use a newer compiler stack - hopefully.